### DISCUSSION TITLE
ColorMode: Forcing a specific mode on Nextjs Pages

### DISCUSSION THREAD

**Discussion by 0x6a68** [chakra-ui/chakra-ui#2526] [Q&A] [answered]
Hi there. First of all congrats to v1 - really enjoying chakra-ui! :confetti_ball: 

I was wondering if it is possible to force a specific ColorMode (dark/light) on a complete Nextjs Page (includes also `styles.global`) and not only on components. reference: https://chakra-ui.com/docs/features/color-mode#forcing-a-specific-mode

**Comment by chasinhues** [comment, marked as answer]
I think you might be able to achieve this by passing config to the theme object as documented here: https://chakra-ui.com/docs/features/color-mode#change-color-mode-behavior

```jsx
const config = {
  useSystemColorMode: false,
  initialColorMode: "light",
}

const customTheme = extendTheme({ config })
```

This should disable the app from using the users default OS colour mode, and always use the initial colour mode that you set. As long as you don't have any triggers in your app that change the colour mode, I imagine this would essentially be the solution for forcing a specific colour mode 😄

**Reply by chasinhues** [reply]
Just to add to this answer - there are some community resources that might come in handy for using Chakra and Next.js:
https://chakra-ui.com/resources
